api/admin/configuration_settings_controller.py

```diff
@@ -0,0 +1,57 @@
+from nose.tools import set_trace
+from core.model import ConfigurationSetting
+from controller import AdminCirculationManagerController
+from api.config import Configuration
+from flask import Response
+from problem_details import *
+import flask
+from flask_babel import lazy_gettext as _
+
+class SitewideConfigurationSettingsController(AdminCirculationManagerController):
+
+    def process_settings(self):
+         self.require_system_admin()
```

Comment:
@aslagle I don't need you to review this whole branch, but I'd like your take on whether require_system_admin() should be called here, or individually in `process_get` and `process_post`. Also, would it be better to eliminate `process_settings` altogether by setting up two different routes in `admin/routes.py`, one for GET and one for POST?

Reply:
It doesn't matter about `require_system_admin` but it's good that it's called before anything else happens. I think it would be good to break up the routes. Almost all the admin routes have the same decorators and many of them also call the same method for GET and POST, but I think that's because the earlier ones had substantial shared code for both cases, and this only has one line.